README.md

```diff
@@ -76,8 +76,13 @@ $ ls opencv2
 ```
 
 
+#### Linux (Ubuntu)
+See links [1](http://docs.opencv.org/trunk/doc/tutorials/introduction/linux_install/linux_install.html) and [2](https://github.com/jayrambhia/Install-OpenCV) on how to install OpenCV `3.2.0`.
+Comment out the #include statements for `flann` in `opencv2/opencv.hpp`.
+
+
```

Comment:
Please remove the entire section below `Windows and Linux`.  We will add notes on support for Windows when it becomes available.  Thanks.